src/graph/graph_hub/update_node_labels.c

```
@@ -0,0 +1,232 @@
+	// attach it to vector v
+	GrB_Info info = GxB_Vector_Iterator_attach (it, _V, NULL) ;
+	ASSERT (info == GrB_SUCCESS) ;
+
+	// seek to the first entry
+	info = GxB_Vector_Iterator_seek (it, 0) ;
+	while (info != GxB_EXHAUSTED) {
+		// get the entry v(i)
+		Node node ;
+		GrB_Index node_id = GxB_Vector_Iterator_getIndex (it) ;
+		Graph_LabelNode (g, node_id, &lbl_id, 1) ;
+
```

Cppcheck (2.20.0)

[error] Memory leak

(memleak)